packages/core/src/components/Links/_Links.docs.scss

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
I'm not sure if it makes sense to group Links under components, as its just an `<a>` tag. Maybe we should consider making the stylings a utility class and moving it under utilities?

Reply:
Actually, I'd even be ok with just having a default styling and not having an additional documentation page. Since its just a native HTML element I dont think we need much guidance.

Reply:
I was thinking it would be nice to have to show what the styling looks like. Maybe under components is not the best place. This brings up the definition of component thing for me and does component mean React component or general HTML component?

Reply:
I think it makes sense if we included it in the typography section! As far as components go, I don't think if its react or html, as long as it's showcasing something that you use in the DS like a set of classes or certain markup.